Documentation/docs-site/content/usage/executor-types.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Executor Types"
+date: 2018-02-01T23:10:05-07:00
+draft: false
+weight: 41
+---
+## Executors and ExecutorType
```

Comment:
I think this doc mixes the user-visible features a bit with the implementation.

How about following document structure

## Overview

same overview

## Executors: How Fission loads runs your function on Kubernetes

[this section should talk a bit about the internals, but not too deeply]

### Pool-based executor

this is your poolmgr section

### New-deployment executor

this is your newdeploy section

### The latency vs. idle-cost tradeoff

Talk about the 3 strategies, minscale>0 / poolmgr / minscale=0, and how they affect cold-start latency and idle memory usage

## Invocation Strategies: Controlling Function Execution  

[this section should be basically a how-to guide for each of these features, including commands that the user can copy-paste]

### Low cold-start latency, low idle cost

poolmgr, defaults

### Eliminating Cold-Start Latency with an idle cost

minscale>0

### Autoscaling

... lots of stuff :)

### Controlling Resource requirements

Suggest explaining a little bit why the executor matters here.

#### Poolmgr executor
#### Newdeploy executor